mysql-test/main/mysql_upgrade.test

```diff
@@ -56,7 +56,7 @@ DROP USER mysqltest1@'%';
 
 --echo Run mysql_upgrade with a non existing server socket
 --replace_result $MYSQLTEST_VARDIR var
```

Comment:
Should the `$MYSQLTEST_*` variables also have `$MARIADBTEST_*` equivalents/aliases?

Or as that getting too much into "internal implementation" as opposed to "user-facing features"?

Reply:
Its a bit internal, and it becomes quite extensive once you start looking. It also starts making merging from lower versions up a more involved process.